akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/ScalaDslUtils.scala

```diff
@@ -8,21 +8,17 @@ import akka.actor.typed.ActorRef
 import akka.persistence.testkit.{ Cmd, CommonUtils, EmptyState, Evt, Passivate, Recovered, Stopped, TestCommand }
 import akka.persistence.typed.PersistenceId
 import akka.persistence.typed.RecoveryCompleted
-import akka.persistence.typed.scaladsl.{ Effect, ReplicatedEventSourcedBehavior }
```

Comment:
Ah, god damn it, IntelliJ did too much and I reverted but missed that it changed here as well. Thx